src/chainparams.cpp

```diff
@@ -134,7 +134,8 @@ class CMainParams : public CChainParams
         nBIP65ActivationHeight = 125000; // Last v3 block was 124712, leave a bit of padding
         nPoAFixTime = 1616716800; // Fork time for PoA fix - Friday, March 26, 2021 12:00:00 AM (GMT)
         nPoAPaddingBlock = 169869; // Last block with 120 PoS blocks in a PoA Audit
-        
+        nPoAPadding = 30; // Current PoA Padding
```

Comment:
We can use 95 to get a padding with 30 PoS Blocks, 95 - 65 max pos count is equal to 30

Reply:
> We can use 95 to get a padding with 30 PoS Blocks, 95 - 65 max pos count is equal to 30

100% Agreed, this one is just to add the current setup to the Chainparams, as we will use it for our next Consensus changes. No functional change, but easier access to variable.

We can make those changes in one big PR for the PoA Consensus Change, setting the 95 and work from there.

Reply:
Nice